src/main/java/hudson/plugins/cmvc/util/CmvcRawParser.java

```diff
@@ -202,7 +201,7 @@ public static CmvcChangeLogSet parseChangeLogFile(Reader xml, CmvcChangeLogSet c
 	 * @throws SAXException
 	 */
 	public static List<CmvcChangeLog> parseChangeLogFile(Reader reader) throws IOException, SAXException {
-		Digester digester = new Digester2();
+		Digester digester = new Digester();

```

Comment:
maybe we should make it safe. (nothing blocker and we are not sure we will release it..)

Reply:
+1 for the safe creation

Reply:
This is a strong requirement actually, not just a nice to have. Or We're going from the Core safe version to the default unsafe one...

Reply:
Yes, I swear I saw below the plugin doing that, but it's not the case, so fixing it.